tests/v3_api/test_project_quota.py

```diff
@@ -0,0 +1,162 @@
+import os
+import pytest
+from .common import *  # NOQA
+
+namespace = {"p_client": None, "ns": None, "cluster": None, "project": None}
+
+CLUSTER_NAME = os.environ.get("CLUSTER_NAME", "")
+RANCHER_CLEANUP_PROJECT = os.environ.get("RANCHER_CLEANUP_PROJECT", "True")
+
+
+@pytest.fixture
+def ns_default_quota():
+    return {"limit": {"pods": "5",
+                      "requestsCpu": "500m"}}
+
+
+@pytest.fixture
+def default_project_quota():
+    return {"limit": {"pods": "20",
+                      "requestsCpu": "2000m"}}
+
+
+def ns_quota():
+    return {"limit": {"pods": "10",
+                      "requestsCpu": "500m"}}
+
+
+def test_create_project_quota():
+    # Create Project Resource Quota and verify quota is created
+    # successfully. Verify namespacedefault resource quota is set
+
+    cluster = namespace["cluster"]
+    client = get_admin_client()
+    c_client = namespace["c_client"]
+
+    quota = default_project_quota()
+    nsquota = ns_default_quota()
+    proj = client.create_project(name='test-' + random_str(),
+                                 clusterId=cluster.id,
+                                 resourceQuota=quota,
+                                 namespaceDefaultResourceQuota=nsquota)
+    proj = client.wait_success(proj)
+
+    assert proj.resourceQuota is not None
+    assert proj.resourceQuota.limit.pods == '20'
```

Comment:
@soumyalj  Could you use quota["pods"]/quota["requestsCpu"] instead of hardcoding the values ?